typewise_alert_data.c

```diff
@@ -0,0 +1,33 @@
+#include "typewise-alert.h"
+
+char SendMessage[100];
+
+const BreachLimitsType BreachLimits[] =
+{

```

Comment:
Maintaining an array for limits of different cooling types could cause errors in future. If lower limits are other than zero, then one more array would be needed and the values at a particular index should be of same cooling type. if the order is not maintained then that could cause problems in the behavior. this can be avoided by using encapsulation concept i.e. keep related data together, either using a struct or class.

Reply:
BreachLimitsType is indeed a struct type. where in lower-limit can be added later in case of new values. @prithamvshetty